db/revision_cache.go

```diff
@@ -112,8 +112,9 @@ type revCacheValue struct {
 }
 
 type RevCacheDelta struct {
-	ToRevID    string
-	DeltaBytes []byte
+	ToRevID       string
+	DeltaBytes    []byte
+	ToAttachments AttachmentsMeta
```

Comment:
Seems a little heavyweight to do this, so maybe we should optimise for this specific fix, and only store a list of digests.

Would appreciate your thoughts @adamcfraser ?

Reply:
Agree that we should probably switch to a slice of strings to ensure we're not accidentally pulling in things like attachment data.